Watchman.Discord/Integration/DevscordFramework/CustomCommandsLoader.cs

```diff
@@ -42,6 +42,9 @@ public async Task InitDefaultCustomCommands()
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.AddRoleCommand", @"-add\s*role\s*(?<Roles>[\w\W\s\""]*)", serverId),
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.RemoveRoleCommand", @"-remove\s*role\s*(?<Roles>[\w\W\s\""]*)", serverId),
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Administration.BotCommands.SetRoleCommand", @"-set\s*role\s*(?<Roles>[\w\W\s\""]*)\s*-((?<Safe>safe)|(?<Unsafe>unsafe))", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.Warns.AddWarnCommand", @"-addwarn\s*-user\s*<@&?\d+>\s*-reason\s*.+", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.Warns.GetWarnsCommand", @"-warns\s*-user\s*<@&?\d+>\s*", serverId)
```

Comment:
nie masz grup w tych regexach, to nie zadziała

Reply:
znaczy jak wpisuję w discordzie komendy to działają, ale jak coś jest źle to chętnie poprawię tylko nie wiem jak, oraz o co chodzi z tymi grupami?

Reply:
no to ci nie ma prawa zadziałać bo nie ma grup

Reply:
jeszcze jedna rzecz
IBotCommand samo z siebie ci generuje to jak ma wyglądać komenda

CustomCommands są po to, żeby dało sie robić - jak nazwa wskazuje, niestandardowe wyglądy komend

np zamiast
`-addrole -roles csharp`
możesz mieć
`-add role csharp`
albo idąc dalej
`daj dla mnie role csharp`